lib/ddtrace/contrib/kafka/events/producer/deliver_messages.rb

```diff
@@ -16,6 +16,7 @@ def self.process(span, _event, _id, payload)
               super
 
               span.set_tag(Ext::TAG_ATTEMPTS, payload[:attempts]) if payload.key?(:attempts)
+              span.set_tag(Ext::TAG_TOPIC, payload[:topic]) if payload.key?(:topic)
```

Comment:
Does the instrumentation for the producer's `deliver_messages` have a topic tag? My understanding is that messages for possibly multiple topics are produced to a buffer, and then those messages for multiple topics are send to Kafka when `deliver_messages` is called.

https://github.com/zendesk/ruby-kafka/blob/8dc495f471312834e477cd16144a32295c6b0a65/lib/kafka/producer.rb#L250-L266